themes/default/content/docs/pulumi-cloud/oidc/_index.md

```diff
@@ -27,6 +29,19 @@ The token contains the standard audience, issuer, and subject claims:
 | `iss` | _(Issuer)_ The issuer of the OIDC token: `https://api.pulumi.com/oidc`.                                                                                                                                                                                                                                                                                                                                                       |
 | `sub` | _(Subject)_ The subject of the OIDC token. Because this value is often used for configuring trust relationships, the subject claim contains information about the associated service. Each component of the subject claim is also available as a custom claim. |
 
+### Pulumi ESC
+
+The token contains the standard audience, issuer, and subject claims:
```

Comment:
Copy pasta?

Reply:
Yep. Will edit to say "The token contains the following claims:"